csrc/ir_internal_nodes.h

```diff
@@ -1938,6 +1938,9 @@ class TORCH_CUDA_CU_API TensorDomain : public Val {
   // pair is in order where second is the consumer of first
   std::pair<TensorDomain*, TensorDomain*> rFactor(const std::vector<int>& axes);
 
+  // clear all transformations and reset to root domain when it was created
+  void resetToRootDomains();
```

Comment:
This is a very dangerous operation as it potentially breaks consistency. Why do you need to do this?

Reply:
`resetToRootDomains` is removed.